### PR TITLE
fix(Image Slider): Use regular buttons for navigation controls

### PR DESCRIPTION
**Pull request by RubenSibon**
# Describe the pull request

Thank you for contributing to the project!
Please use this template to help us handle your PR smoothly.

## What

Replaces the IconButton with a normal Button that has the `iconOnly` property set.

Also refactors the stylesheet to conditionally render the buttons depening on form factor in a mobile-first way.

## Why

It was not according to the design guidelines.

## Checklist

Before submitting your pull request, please ensure you have done the following. Check each checkmark if you have done so or if it wasn't necessary:

- [x] Add or update unit tests
- [x] Add or update documentation
- [x] Add or update stories
- [x] Add or update exports in index.\* files
- [x] Start the PR title with a Conventional Commit prefix, [as explained here](https://github.com/Amsterdam/design-system/blob/main/documentation/publishing.md?plain=1#L11).

## Additional notes

- Provide links to any related issues or discussions.
- Add a link to the specific story in the feature branch deploy.
- Mention any areas where additional review or feedback is needed.

### CODE REVIEW

File: packages/css/src/components/image-slider/image-slider.scss [hunk 1]
```diff
@@ -45,13 +45,13 @@
...
 .ams-image-slider__controls {
-  display: flex;
+  display: none;
   grid-column: 1/-1;
   grid-row: 1;
   justify-content: space-between;
 
-  @media (pointer: coarse) and (max-width: $ams-breakpoint-medium) {
-    display: none;
+  @media (min-width: $ams-breakpoint-medium) {
```
Comment (dlnr) [inline packages/css/src/components/image-slider/image-slider.scss:53]: Why did you change this? It's not working for me, I get no buttons on a non-touch screen desktop. The idea was to make a small edge case to hide the controls on non mouse devices under the tablet size. This was discussed during the build of the image slider and not in the scope of this ticket.

Comment (RubenSibon) [reply packages/css/src/components/image-slider/image-slider.scss:53]: Haven you check the latest deploy? I pushed a commit to fix the non-touch screen desktop part.

Comment (RubenSibon) [reply packages/css/src/components/image-slider/image-slider.scss:53]: It was requested by Vincent: https://gemeente-amsterdam.atlassian.net/browse/DES-1156?focusedCommentId=439178

Comment (VincentSmedinga) [reply packages/css/src/components/image-slider/image-slider.scss:53]: I think @dlnr is referring to the `pointer: coarse` condition :)

Comment (RubenSibon) [reply packages/css/src/components/image-slider/image-slider.scss:53]: But then there are no buttons on larger screens that are not touch enabled.

Comment (VincentSmedinga) [reply packages/css/src/components/image-slider/image-slider.scss]: > But then there are no buttons on larger screens that are not touch enabled.

Oooh wait I had’t realised that 🤦🏼‍♂️

We can inverse the conditions, right?

```
// We show controls on non-touch devices, where people can’t swipe the slides,
// but not in narrow windows, where they obscure too much of an image.
@media (min-width: $ams-breakpoint-medium) and (pointer: fine) {
  display: flex;
}
```

Comment (VincentSmedinga) [reply packages/css/src/components/image-slider/image-slider.scss]: Updated `pointer: fine` to `not pointer: coarse` to not exclude screen readers and keyboard users.

Comment (RubenSibon) [reply packages/css/src/components/image-slider/image-slider.scss]: Yeah, this works, but will this show the buttons on touch-enabled laptops or hybrid tablets with a keyboard (i.e. Surface-type devices)? I think so, but I'm not really sure. What's the quickest and most reliable way to test that?

Comment (RubenSibon) [reply packages/css/src/components/image-slider/image-slider.scss]: Are those devices too much of an edge case to worry about? Those users can still use the thumbnails or swiping.

Comment (VincentSmedinga) [reply packages/css/src/components/image-slider/image-slider.scss]: The [Polypane](https://polypane.app) browser offer lots of emulation options 💪🏼 

And yes, there are many possible combinations and the existing approach addressed some of them. If I was using a laptop with touch screen, I’d still reach for the mouse to navigate between the slides, I think – but I’d expect the buttons to be invisible on these devices with this media query.

We should probably just update the current media query to work mobile-first like all others, and consider other use cases when issues arise.

File: packages/css/src/components/image-slider/image-slider.scss [hunk 2]
```diff
@@ -45,13 +45,15 @@
...
   grid-row: 1;
   justify-content: space-between;
 
-  @media (pointer: coarse) and (max-width: $ams-breakpoint-medium) {
-    display: none;
+  // We show controls on non-touch devices, where people can’t swipe the slides,
+  // but not in narrow windows, where they obscure too much of an image.
+  @media (min-width: $ams-breakpoint-medium) and (pointer: fine) {
+    display: flex;
   }
```
Comment (VincentSmedinga) [inline packages/css/src/components/image-slider/image-slider.scss:57]: I updated the suggestion in my comment a bit later, but apparently you’d already copied the original one. Once more:

```suggestion
  // We hide controls on touch devices, where people can swipe the slides,
  // and also in narrow windows, where they obscure too much of an image.
  @media (min-width: $ams-breakpoint-medium) and (not (pointer: coarse)) {
    display: flex;
  }
```

Comment (RubenSibon) [reply packages/css/src/components/image-slider/image-slider.scss:57]: Ah, interesting. I'll try it out.

Comment (RubenSibon) [reply packages/css/src/components/image-slider/image-slider.scss:57]: Tested it: in the device simulation in Chromium's devtools the buttons are not visible on any hybrid devices such as the Microsoft Surface devices and the Asus Zendbook Fold.

**Review by VincentSmedinga — APPROVED** [review]
Sorry for the detour! :)
